app/assets/js/script.js

```diff
@@ -4,6 +4,11 @@
   var $win = $(window);
   var $doc = $(document);
 
+  // For assetgraph to pick up strings as urls
```

Comment:
It can be added as a separate script include with `id="bootstrap"`, which the builer will take as an indication to clean it out during build